src/main/java/lekcja19_zadanie2_poprawnosc_hasla/Password.js

```diff
@@ -0,0 +1,37 @@
+const password = document.getElementById("exampleInputPassword");
+const confirmPassword = document.getElementById("confirmInputPassword");
+const errorMessage = document.getElementById("error");
+const confirmPasswordErrorMessage = document.getElementById("cerror");
+const form = document.getElementById("form");
+const input = document.getElementsByTagName("input")
+
+form.addEventListener('input', ev => {
+    let passwordMessages = [];
+    let confirmPasswordMessages = [];
+
+    if (password.value.length < 7) {
+        passwordMessages.push('Za krótkie hasło. Min. 8.');
+    }
+
+    if (!password.value.match(/[A-Z]/)) {
+        passwordMessages.push('Min 1 duża litera.');
+    }
+
+    if (!password.value.match(/[`!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~]/)) {
+        passwordMessages.push('Min 1 znak specjalny.');
+    }
+
+    if (password !== confirmPassword) {
+        confirmPasswordMessages.push('Hasła różnią się od siebie!');
+    }
+
+    if (passwordMessages.length > 0) {
```

Comment:
Wiadmości się nie kasują, z powodu tego kawałka kodu - jeżeli jest wszystko dobrze to nie usuwasz starych wiadomości a zostawiasz nowe.

Reply:
poprawione